### PR TITLE
Develope branch

### PR DESCRIPTION
**Pull request by DmitrySidorov**
Initial Pull-Request with main business logic implementation for the numeric Matrix utility class.

### CODE REVIEW

**Comment by DmitrySidorov** [comment]
Hi Ilya, generally looks good for the first time! Please refer to my comments, and merge it to the master branch. Thanks!

**Comment by DmitrySidorov** [comment]
Several other remarks here:

1) Spelling mistake
image

2) Avoid names like this with a dash at the end
image

3) These files shouldn't be here
image

**Comment by DmitrySidorov** [comment]
useful info

https://www.atlassian.com/ru/git/tutorials/saving-changes/gitignore

File: Program.cs [hunk 1]
```diff
@@ -7,18 +7,16 @@ class Matrix
     public int rows;
     public int cols;
 
-    public void Read()
-    {
-        //Console.WriteLine("Type an range of your matrix here: " );
-        var range = (Console.ReadLine().Split(" ")); //     2 3
+    public void Read()// creating an objoct of Matrix
+    {        
+        var range = (Console.ReadLine().Split(" "));
```
Comment (ellia-w) [inline Program.cs:12]: A new comment here.

**Comment by DmitrySidorov** [comment]
image

**Comment by DmitrySidorov** [comment]
image

**Comment by DmitrySidorov** [comment]
image

**Pull request merged**